Sming/Arch/Host/Platform/AccessPointImpl.cpp

```diff
@@ -53,7 +53,7 @@ bool AccessPointImpl::setIP(IPAddress address)
 	return false;
 }
 
-MACAddress AccessPointImpl::getMacAddr() const
+MACAddress AccessPointImpl::getMacAddress() const
```

Comment:
Please, rename `MACAddress` to `MacAddress`. Since it is new no need to add any notes to the migration document.